action.yml

```diff
@@ -12,11 +12,15 @@ inputs:
     description: The Input path.
     default: '.'
     required: false
+  enable_draft:
+    description: The identifier of enabling draft, when set to 'true' the action will push the module as draft if the branch that trigger the workflow is not 'main'.
+    required: false
```

Comment:
Should this default to true? That would be a behavior change, but seems like the expected ux? Or do we want to punt on that till the sync story is more clear and let this be off by default?

Reply:
same with the comment about behavior change  above, I can change this if you think it is more reasonable